### PR TITLE
fix(web): batch realtime entry creates during catch-up

### PR DESCRIPTION
**Pull request by iradkot**
## What

- Buffer short bursts of realtime `entries` create events and apply them in one reactive update.
- Preserve the existing ID and `(mills, sgv)` deduplication, newest-first ordering (including equal timestamps), and 1,000-entry cap.
- Treat the expected zero-Socket.IO-client state as debug information instead of emitting one warning per backfilled event.
- Add regression tests for a sustained 100-reading catch-up burst, duplicate readings with different IDs, and equal-timestamp tie ordering.

## Why

Connector catch-up sends each imported reading as an individual Socket.IO create event. The store previously copied and sorted the entire entries array for every message. Each assignment also retriggered chart derivations, so a few hundred imported readings could make the Readings report unresponsive even while the host was otherwise idle.

I reproduced this with a 497-entry catch-up. Before the change, the report's DOM did not settle within 20 seconds and the bridge emitted dozens of expected no-client warnings per second. With the batched store deployed, fresh Readings loads settled in roughly 0.68–1.15 seconds.

## Verification

- `vitest run --config vitest.config.ts --maxWorkers=1 --no-file-parallelism src/lib/stores/realtime-store.test.ts`: 16 passed
- `pnpm --filter @nocturne/bridge test`: 51 passed
- `pnpm --filter @nocturne/bridge build`: passed
- `pnpm --filter @nocturne/bot build`: passed
- Release API build with generated clients: 0 errors
- `git diff --check`: passed

The full SvelteKit production build reached application transformation but could not finish on this local host after it exhausted available memory. No application compile error was reported before the resource failure.

### CODE REVIEW

File: src/Web/packages/app/src/lib/stores/realtime-store.svelte.ts [hunk 1]
```diff
@@ -596,6 +595,66 @@ export class RealtimeStore {
...
+      if (typeof entry._id === "string") knownIds.add(entry._id);
+      knownReadings.add(readingIdentity);
+      return true;
+    });
+
+    if (additions.length > 0) {
+      this.entries = [...this.entries, ...additions]
+        .sort((a, b) => (b.mills || 0) - (a.mills || 0))
+        .slice(0, 1000);
+    }
```
[the remark this answers is not in the document]
Comment (iradkot) [reply src/Web/packages/app/src/lib/stores/realtime-store.svelte.ts:655]: Fixed in 8dca699e. Batch additions are now reversed and prepended before the stable mills sort, matching the previous per-create tie behavior. A regression test asserts second-create, first-create, existing for identical mills values. The focused suite passes all 16 tests.
